app/src/main/java/com/example/reflector_android/network/Article.kt

```diff
@@ -2,4 +2,4 @@ package com.example.reflector_android.network
 
 import java.time.LocalDate
 
-data class Article(val title: String?, val description: String?, val pubDate: LocalDate?, val link: String?, val author: String? )
+data class Article(val title: String?, val description: String?, val pubDate: LocalDate?, val link: String?, val author: String?, val isLoading: Boolean?)
```

Comment:
I highly advise against adding isLoading to the Article class at all possible because our data should be 100% separate from stuff that only pertains to our View code. isLoading is not part of the RSS feed <item> models so we also should not have isLoading.

If you need to have a flag to keep up with if you are loading something, this flag is often kept inside your View code or ViewModel code.